src/pages/Notes/EditNote.tsx

```diff
@@ -39,8 +40,9 @@ export const EditNote = ({ note }: Props) => {
         setNotes((old) => [...old.filter((n) => n.id !== note.id), response.note]);
       })
       .catch((response) => {
+        const apiProblem = parseError(response);
         console.error('Not ok!');
-        console.debug(response.status, response.ok);
+        console.debug(apiProblem.title, apiProblem.detail, apiProblem.status);
```

Comment:
```suggestion
        console.warning(apiProblem.title, apiProblem.detail, apiProblem.status);
```

Just a tiny imporvement